cli/root.go

```diff
@@ -127,12 +126,20 @@ func Execute() {
 	app.Flag("alertmanager.url", "Alertmanager to talk to").URLVar(&alertmanagerURL)
 	app.Flag("output", "Output formatter (simple, extended, json)").Short('o').Default("simple").EnumVar(&output, "simple", "extended", "json")
 	app.Flag("timeout", "Timeout for the executed command").Default("30s").DurationVar(&timeout)
-	app.Flag("tls.insecure.skip.verify", "Skip TLS certificate verification").BoolVar(&tlsInsecureSkipVerify)
+	app.Flag("tls.certfile", "TLS client certificate file").PlaceHolder("client.crt").ExistingFileVar(&tls.CertFile)
+	app.Flag("tls.keyfile", "TLS client private key file").PlaceHolder("client.key").ExistingFileVar(&tls.KeyFile)
+	app.Flag("tls.cafile", "TLS trusted certificate authorities file").PlaceHolder("ca.crt").ExistingFileVar(&tls.CAFile)
```

Comment:
Can you remove the files placeholders? They look like default values in --help, I don't like it.

Reply:
Sure. I wasn't sure what to do with these. I didn't really like it either way. The placeholders shout more, but at least they don't look like filenames

Reply:
Oh
I have inspiration

Reply:
In prometheus, the placeholders are like `<url>`, `<path>` (url path), we could do `<file>` for the files. And fix the URL.

Reply:
Let's take the placeholders from https://prometheus.io/docs/prometheus/latest/configuration/configuration/

<filename> and <string> for the servername.